github_actions/lib/dependabot/github_actions/file_parser.rb

```diff
@@ -30,12 +30,6 @@ def parse
           dependency_set += workfile_file_dependencies(file)
         end
 
-        dependencies_without_version = dependency_set.dependencies.select { |dep| dep.version.nil? }
-        unless dependencies_without_version.empty?
-          raise UnresolvableVersionError,
```

Comment:
I think this is the only portion to be reverted but for clarity and simplicity; feel free to go ahead.